### PR TITLE
Fix a build break from code merge issue.

### PR DESCRIPTION
**Pull request by yizhang4321**
Two previous PRs went in resulted in this break.

### CODE REVIEW

File: pkg/agent/utils.go [hunk 1]
```diff
@@ -850,7 +850,7 @@ func IsDynamicKubeletEnabled(cs *datamodel.ContainerService, dynamicKubeletToggl
 
 func ensureKubeletConfigFlagsValue(kc map[string]string, cs *api.ContainerService, dynamicKubeletToggleEnabled bool) {
```
Comment (bowang-666) [inline pkg/agent/utils.go:851]: could just change the signature of this func to take *datamodel.ContainerService

Comment (yizhang4321) [reply pkg/agent/utils.go:851]: Yes, I'll make the change.

Comment (yizhang4321) [reply pkg/agent/utils.go]: Code updated.

**Pull request merged**